src/Phinx/Config/Config.php

```diff
@@ -60,9 +60,9 @@ public function __construct(array $configArray, $configFilePath = null)
      *
      * @param string $configFilePath Path to the Yaml File
      * @throws \RuntimeException
-     * @return \Phinx\Config\Config
+     * @return \Phinx\Config\ConfigInterface
```

Comment:
Weakened this to `\Phinx\Config\ConfigInterface` as I believe if someone extends off `Config` and runs this function, then it would return an instance of the extended class and not `Config` via the usage of the `static` keyword on initializing the class.